aws/resource_aws_subnet.go

```diff
@@ -72,6 +72,12 @@ func resourceAwsSubnet() *schema.Resource {
 				Default:  false,
 			},
 
+			"outpost_arn": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
can you add the following to allow validation the ARN
```				
ValidateFunc: validateArn,
```

Reply:
Added